src/test/resources/Leas_Liudmila/The first Feature file.feature

```diff
@@ -0,0 +1,28 @@
+Feature: The first automation tests
+
+  Scenario: Average of three numbers
+
+    Given I have three numbers "5", "4" and "6"
+    When I find average of three numbers
+    Then the average should be "5"
+
+    # 1: Matching Digits: Write a regex pattern that matches any sequence of digits in a given string.
+    #  Test it with various inputs, such as "Hello123", "456", and "abc789xyz".
+
+  # Result:
+  # [231|654|879]
+  # [0-9]{1,20}
```

Comment:
1,20 will find max 20 digints, I hope you understand 
I think + will be ok instead, the same as {1, }